README.md

```diff
@@ -41,7 +41,10 @@ virtual machine:
  * Run `vagrant plugin install vagrant-registration` to install plugin
 * [vagrant-adbinfo plugin](https://github.com/bexelbie/vagrant-adbinfo) _(>=0.0.9)_ installed
  * Run `vagrant plugin install vagrant-adbinfo` to install plugin
-
+* On Windows:
+ * Ensure putty utilities, including pscp, are installed and on the Path
```

Comment:
I don't think putty is a requirement? I don't have it installed and it works fine. cygwin w/ rsync and openssh yes. This ties a bit into the discussion here - https://github.com/redhat-developer-tooling/openshift-vagrant/issues/24

In which "shell" am I supposed to use Vagrant on Windows?